examples/hosts/app-integration/external-data/src/view/debugView.tsx

```diff
@@ -122,20 +109,16 @@ interface ISyncStatusViewProps { }
 const SyncStatusView: React.FC<ISyncStatusViewProps> = (props: ISyncStatusViewProps) => {
     return (
         <div>
-            <h3>Sync status</h3>
+            {/* <h3>Sync status</h3>
```

Comment:
Probably keep this?  @WayneFerrao has been working in this space so I think he would have to add it back.